### ISSUE TITLE
Documentation proposal: Be upfront about limitations

### ISSUE THREAD

**Issue by jmn** [containous/traefik#1049]
While Traefik does do a bunch of cool and interesting stuff, I suggest adding a "limitations" section on the Readme.md which states what Traefik currently does not do and/or does in a limited fashion. This would quickly let users see if this the right tool for them and could save people a lot of time I think. Noting if work in that area is out of scope or PR's wanted would also be useful. E.g: 

## Limitations:
- Traefik currently **does not** proxy TCP (see nginx, haproxy)

**Comment by sheerun** [comment]
Yes, I discovered it too late as well :) I use it anyway, but it should be documented

**Comment by linuxus** [comment]
Yup... this became a show stopper for me as well. Great suggestion tho!
